sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/CBDE/MlirExportTest.cs

```diff
@@ -1226,6 +1226,11 @@ public void f(int i)
     i *= 3;
     i /= 4;
     i %= 5;
+    i >>= 6;
+    i <<= 7;
```

Comment:
Since this case is more complex than usual, I think it might deserve a test in `CheckMlirWithReference`